src/hooks/useHidden.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  UseInputParameters,
+  BasicInputReturnType,
+  UseRawInputReturnType,
+} from './types'
+import { useGenericInput } from './useGenericInput'
+import { getRawInputCustomFields } from './useRawInput'
+
+export const getHiddenCustomFields = (
+  baseObject: BasicInputReturnType
+): UseRawInputReturnType => {
+  baseObject.isRequired = false
+
+  const returnObject = getRawInputCustomFields(baseObject, 'hidden')
+
+  returnObject.getLabelProps = () => {
+    return {}
+  }
```

Comment:
What is this?

Reply:
I clear the getLabelProps() function returned from the object as there can't be a label referring to a hidden field